ci/cmd/maestro/kubernetes_tools.go

```diff
@@ -39,7 +39,9 @@ func GetPodLogs(kubeClient kubernetes.Interface, namespace string, podName strin
 		os.Exit(1)
 	}
 
-	defer logStream.Close()
+	defer func() {
+		_ = logStream.Close()
+	}()
```

Comment:
In such cases, if you are disregarding error checks anyway, you can annotate the linter to not flag such errors by using `//nolint: ...`.